src/packages/utils/__init__.py

```diff
@@ -10,21 +10,24 @@
 """
 
 from .autocorrect import correct_system_name
+
+from .starter_systems import isStarterSystem
 from .ratlib import sanitize, Vector, Colors, color, bold, underline, italic, reverse, Platforms, \
     Singleton, Status, Formatting
 
 __all__ = [
-    "autocorrect",
-    "ratlib",
+    "correct_system_name",
+    "isStarterSystem",
```

Comment:
[style] PEP8 [requires `snake_case` naming for functions and methods](https://www.python.org/dev/peps/pep-0008/#id45).